pucauto.py

```diff
@@ -10,10 +10,21 @@
 from bs4 import BeautifulSoup
 from lib import logger
 
-
-with open("config.json") as config:
-    CONFIG = json.load(config)
-
+try:
+    import config
+except:
+    print 'No config.py found!'
+    exit()
+
+config_vars = dir(config)
+for var in ['username', 'password', 'min_value']:
```

Comment:
I'd also prefer to remove this and just let the script blow up if they don't have a valid config.py. Less code. Not my fault they didn't match the example. :stuck_out_tongue: